src/app/components/markets/MarketsInfo.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+
+export function MarketsInfo() {
+  const marketPairs = 0;
+  const xrdVolume = 0;
+  const volumeChange = 0;
```

Comment:
these 0s do not look right, shouldn't we update them from somewhere?
<img width="1450" alt="Screenshot 2023-11-15 at 19 30 37" src="https://github.com/DeXter-on-Radix/website/assets/27793901/0a4ff0b0-abf4-4c78-9253-5cd86d8f14a1">

Reply:
@fliebenberg would the alphadex api have these numbers somewhere or would we count/add this up on the client side?

Reply:
Not at the moment, but I will add an API point to provide this kind of info. Am working on the API at the moment, so will add it and let you know once done. Hopefully today still.

Reply:
I have added a new endpoint to the AlphaDEX api at /alphadex/market_info. It returns an object with the following fields:
- noPairs:  the number of pairs on the exchange
- noTokens: the number of tokens on the exchange
- xrdValueOfLiquidity: the XRD value of the total liquidity provided on the exchange
- noTrades24h: the number of completed trades on the exchange in the last 24h
- xrdValueOfTrades24h: the XRD value of completed trades in the last 24h

Let me know if this works for you.